api/scpca_portal/test/management/commands/test_create_portal_metadata.py

```diff
@@ -1,41 +1,46 @@
 import csv
 import shutil
+from unittest.mock import patch
 from zipfile import ZipFile
 
 from django.test import TransactionTestCase
 
 from scpca_portal import common, metadata_file, readme_file
-from scpca_portal.management.commands import create_portal_metadata, load_data
 from scpca_portal.models import ComputedFile, Library, Project, Sample
 
+from scpca_portal.management.commands import create_portal_metadata  # isort:skip
+from scpca_portal.management.commands import configure_aws_cli  # isort:skip
+from scpca_portal.management.commands import load_data  # isort:skip
+
 # NOTE: Test data bucket is defined in `scpca_porta/common.py`.
 # When common.INPUT_BUCKET_NAME is changed, please delete the contents of
 # api/test_data/input before testing to ensure test files are updated correctly.
 
 ALLOWED_SUBMITTERS = {"scpca"}
 
+README_FILE = readme_file.OUTPUT_NAME
+METADATA_FILE = metadata_file.MetadataFilenames.METADATA_ONLY_FILE_NAME
+
 
 class TestCreatePortalMetadata(TransactionTestCase):
     def setUp(self):
         self.processor = create_portal_metadata.Command()
         self.loader = load_data.Command()
+        configure_aws_cli.Command()  # TEMP Prevent load_data test from running in parallel
```

Comment:
This is the line to prevent tests from running in parallel. 

Additionally, I've temporarily disabled `isort` from modifying the import order of the management commands. As mentioned in the eng meeting, the import for `create_portal_metadata` must come before `configure_aws_cli` to avoid this specific issue. 

I was thinking of removing the duplicate logging messages to clean up the logs, but I’m not sure if that’s the best move. Please let me know if you have any insights or feedback on this, thank you David!

Reply:
This can just be removed. As per our conversation

Reply:
This has been removed at baa659c0c1a8ec117f7506944ad679a748df2ac7, thank you for the insights, David!